packages/@coorpacademy-components/src/atom/chips/index.js

```diff
@@ -5,16 +5,20 @@ import {
   NovaSolidRemoveAddAddCircle1 as AddIcon,
   NovaSolidStatusCheckCircle2 as SelectedIcon
 } from '@coorpacademy/nova-icons';
+import get from 'lodash/fp/get';
+import Provider from '../provider';
 import style from './style.css';
 
-const Chips = props => {
+const Chips = (props, context) => {
```

Comment:
ça n'a pas d'implication sur l'utilisation du composant @youssefezzahi96 ?

:gb: any effect on the way the Component is to be set up?
Do we need to adapt existing usage of the Chips?

Reply:
I checked the other components that use Chips, no effect